src/server.c

```diff
@@ -6941,6 +6945,9 @@ int main(int argc, char **argv) {
         serverLog(LL_NOTICE, "Server initialized");
         aofLoadManifestFromDisk();
         loadDataFromDisk();
+        del_stat stat = delUnOwnedKeys();
```

Comment:
We should do this later, after "verifyClusterConfigWithData". We want to give this node a chance at taking ownership of this slot.